core/nwb.image.yaml

```diff
@@ -38,8 +38,9 @@ groups:
   neurodata_type_inc: TimeSeries
   doc: General image data that is common between acquisition and stimulus time series.
     Sometimes the image data is stored in the file in a raw format while other
-    times it will be stored as an external image file in the host file system. The
-    data field will either be binary data or empty. [frame][y][x] or [frame][z][y][x].
+    times it will be stored as a series of external image files in the host file system.
```

Comment:
It seems like the original description had something like tif stacks in mind, which is a common way to store acquired data and saves all of the images in a single file, like a video.